docs/04-guides/playback-a-video-stream.mdx

```diff
@@ -6,16 +6,166 @@ description: Playback video and livestreams
+play assets. It provides a core vanilla JS library to easily connect to a
+Livepeer provider (e.g. Livepeer Studio), as well as React-specific
+hooks/components to provide memoization and DOM management.
+
+:::note
+
+Livepeer.js currently only supports applications built with React.js and
+Next.js.
+
+:::
+
+### Get video playback id
+
+Each video or live stream has a unique playback id. You can get the playback id
+from the response of the [Create a video](/reference/api/create-asset-with-url)
+or [Create a livestream](/reference/api/create-stream) API.
+
+The response should look like this:
+
+```js
+{
+    "asset": {
+        "id": "7d042176-230b-464c-a677-4a19b513193c",
+        // highlight-start
+        "playbackId": "7d048tbcfrv6gvzs",
+        // highlight-end
+        "userId": "80dc8f6e-69d5-401f-bbd7-bfc09a2a5320",
+        "createdAt": 1659715086322,
+        "status": {
+            "phase": "waiting",
+            "updatedAt": 1659715086322
+        },
+        "name": "Example name"
+    },
+    "task": {
+        "id": "d3a2ab1b-8dd1-450d-ac8e-498fd9d91865"
+    }
+}
+```
+
+### Installation
+
+Once you have the playback id, the next step is to install the library. You can
+do so by running the following command:
+
+<Tabs>
+  <TabItem value="npm" label="npm">
+    <CodeBlock language="bash">npm install @livepeer/react</CodeBlock>
+  </TabItem>
+  <TabItem value="yarn" label="yarn">
+    <CodeBlock language="bash">yarn add @livepeer/react</CodeBlock>
+  </TabItem>
+</Tabs>
+
+### Usage
+
+The Player component provides an easy way to display video or audio. You can
+also customize the player's theme and controls.
+
+```jsx
+import { Player } from "@livepeer/react";
+
+export const SimplePlayer = () => {
+  return (
+    <Player
+      title="Agent 327: Operation Barbershop"
+      playbackId="6d7el73r1y12chxr"
+      showPipButton
+    />
+  );
+};
+```
+
+<Player />
+
+## Using your own player
+
+Using livepeer.js is the easiest and recommended way to playback a video or a
+live stream. However, if you want to use your own player, you can do so by
+following the instructions below.
+
+:::caution
+
+Please note that to playback live streams inside your application you'll need to
+use a video player component that supports the HLS.
+
+:::
+
+### Create a playback URL
+
+To playback a live stream in other players, you'll need to create an HLS URL.
+HLS is a protocol that allows you to stream video and audio content over HTTP.
+Most of the video you watch on the web is delivered using HLS. Livepeer also
+uses HLS to deliver video and audio content.
+
+You can create a HLS url by using the below format and replacing the
+`PLAYBACK_ID` with your stream's playback id.
+
+```txt
+https://livepeercdn.com/hls/{PLAYBACK_ID}/index.m3u8
+```
+
+However, if you want to playback a video that is already uploaded to Livepeer,
+you can use the below format and replace the `PLAYBACK_ID` with your video's
+playback id.
```

Comment:
I think it would be clearer like this, wdyt?:
```suggestion
On-demand videos have a different format for the playback URLs. To playback such videos,
you can use the below format and replace the `PLAYBACK_ID` with your video's
playback id.
```
(this should be different if you make the remark about the different playback URLs before giving any exampls)

Reply:
sounds great!